cluster/gce/gci/configure.sh

```diff
@@ -110,16 +110,33 @@ function download-or-bust {
   done
 }
 
+function is-preloaded {
+  local -r key=$1
+  local -r value=$2
+  if [[ -f ${KUBE_HOME}/preload_info ]] && (grep "${key}" "${KUBE_HOME}/preload_info" | grep "${value}" > /dev/null 2>&1);then
+    echo 0
```

Comment:
We really should just return 0 or 1 and not write output that has to be captured and later compared. This would work better

```shell
function is-preloaded {
  local -r key=$1
  local -r value=$2
  grep -qs "${key},${value}" "${KUBE_HOME}/preload_info"
}

...

if is-preloaded "mounter" "${mounter_tar_sha}"; then
  echo "mounter is preloaded."
  return
fi
```